### PR TITLE
Allow `oc adm release info` to output digest, name, or pull spec

### PR DESCRIPTION
**Pull request by smarterclayton**
Useful for scripting and automation.

* `-o digest`: outputs the digest, if set, of any images
* `-o pullspec`: outputs the pullspec of referenced images
* `-o name`: outputs the name of the tag of a referenced image

Used this to find all images to verify RPM db from

### CODE REVIEW

**Comment by smarterclayton** [comment]
/cherrypick release-4.1

To enable scripting and tooling.

File: pkg/oc/cli/admin/release/info.go [hunk 1]
```diff
@@ -306,13 +308,42 @@ func (o *InfoOptions) describeImage(release *ReleaseInfo) error {
...
+					fmt.Fprintf(o.ErrOut, "error: %s does not point to a digest\n", tag.Name)
+				} else {
+					fmt.Fprintf(o.Out, "%s\n", ref.ID)
+				}
+			}
+		}
+		return nil
+	case "":
+	default:
+		return fmt.Errorf("only 'json' is supported for output mode")
```
Comment (soltysh) [inline pkg/oc/cli/admin/release/info.go]: `only json, name, pullspec or digest are supported output mode` or just `unsupported output mode %q`.

**Comment by smarterclayton** [comment]
Fixed

**Pull request merged**